File: 10/elixir/main.exs
{item1, _} = Float.parse IO.gets "Enter the price of item 1\n"
{quant1, _} = Integer.parse IO.gets "Enter the quantity of item 1\n"
{item2, _} = Float.parse IO.gets "Enter the price of item 2\n"
{quant2, _} = Integer.parse IO.gets "Enter the quantity of item 2\n"
{item3, _} = Float.parse IO.gets "Enter the price of item 3\n"
{quant3, _} = Integer.parse IO.gets "Enter the quantity of item 3\n"

subtotal = Float.round(((item1 * quant1) + (item2 * quant2) + (item3 * quant3)), 2)
tax = Float.round((subtotal * 0.055), 2)
total = Float.round((subtotal + tax), 2)

IO.puts "Subtotal: $#{Float.to_string(subtotal, [decimals: 2])}"
IO.puts "Tax: $#{Float.to_string(tax, [decimals: 2])}"
IO.puts "Total: $#{Float.to_string(total, [decimals: 2])}"

File: 10/clojure/main.clj
(defn tax [n]
  (* n 0.055))

(println "Enter the price of item 1:")
(def one (Integer. (re-find #"\d+" (read-line))))
(println "Enter the quantity of item 1:")
(def one-quant (Integer. (re-find #"\d+" (read-line))))
(println "Enter the price of item 2:")
(def two (Integer. (re-find #"\d+" (read-line))))
(println "Enter the quantity of item 2:")
(def two-quant (Integer. (re-find #"\d+" (read-line))))
(println "Enter the price of item 3:")
(def tre (Integer. (re-find #"\d+" (read-line))))
(println "Enter the quantity of item 3:")
(def tre-quant (Integer. (re-find #"\d+" (read-line))))
(def subtotal (+ (* one-quant one) 
                 (* two-quant two)
                 (* tre-quant tre)))

(printf "Subtotal: $%.2f%n" (double subtotal))
(printf "Tax: $%.2f%n" (tax subtotal))
(printf "Total: $%.2f%n" (+ subtotal (tax subtotal)))
